xata/utils.go

```diff
@@ -177,11 +177,22 @@ func getBranchName() string {
 		}
 	}
 
-	if branchName, found := myEnv[branchNameEnvVar]; found {
-		return branchName
+	if val, found := myEnv[name]; found {
+		return val
 	}
+	return defaultValue
+}
 
-	return defaultBranchName
+// getBranchName retrieves the branch name.
+// If not found, falls back to defaultBranchName
+func getBranchName() string {
+	return getEnvVar(branchNameEnvVar, defaultBranchName)
+}
+
+// Get the region if the corresponding env var `XATA_REGION` is set
+// otherwise return the default region: us-east-1
+func getRegion() string {
+	return getEnvVar("XATA_REGION", defaultRegion)
```

Comment:
nit: We appear to sometimes uses the raw string value of the environment variable names and sometimes constants. I think we should switch to constants everywhere. I'm happy to do this as a separate PR as doing it as part of this one could get messy.

Reply:
that's a good nit, I felt guilty not doing it :) I'll push it up in a sec and share the commit.

Reply:
https://github.com/xataio/xata-go/pull/54/commits/52291e0a0da1970c3cbe9260ef44c8409ee26c72

Reply:
We still use the string values in a lot of tests. To get around this we'd have to make the constants public.